src/core/highlight-vars.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * Module core/highlight-vars
+ * Highlights occurrences of a <var> within a section on click.
+ * Set `conf.highlightVars = true` to enable.
+ * Removes highlights from <var> if clicked anywhere else.
+ * All is done while keeping in mind that exported html stays clean
+ * on export.
+ */
+import { sub } from "core/pubsubhub";
+import hlVars from "deps/text!core/css/var.css";
+
+export const name = "core/highlight-vars";
+
+export function run(conf) {
+  if (!conf.highlightVars) {
+    return;
+  }
+  const styleElement = document.createElement("style");
+  styleElement.textContent = hlVars;
+  styleElement.classList.add("removeOnSave");
+  document.head.appendChild(styleElement);
+
+  [...document.querySelectorAll("var")].forEach(varElem =>
+    varElem.addEventListener("click", hightlightListener)
+  );
+
+  // remove highlights, cleanup empty class/style attributes
+  sub("beforesave", outputDoc => {
+    [...outputDoc.querySelectorAll("var.respec-active")].forEach(
+      removeHighlight
+    );
+  });
+}
+
+function hightlightListener(ev) {
+  ev.stopPropagation();
+  const { target: varElem } = ev;
+  const hightligtedElems = highlightVars(varElem);
+  const resetListener = () => {
+    hightligtedElems.forEach(removeHighlight);
+    [...HL_COLORS.keys()].forEach(key => HL_COLORS.set(key, true));
+  };
+  if (hightligtedElems.length) {
+    document.body.addEventListener("click", resetListener, { once: true });
```

Comment:
Had a small play on iOS... "click" only works on certain elements, but not the body... and "touchstart" catches all touches. Not sure what the best solution is here for mobile.

Reply:
Can this be a solution too? : https://stackoverflow.com/questions/25333738/click-events-not-firing-on-ios (weird bug, weird fix)